README.md

```diff
@@ -1,42 +1,74 @@
-# EPIC_MAIL
+[![Build Status](https://travis-ci.org/jennizalwango/epicmail2.svg?branch=ft-delete)](https://travis-ci.org/jennizalwango/epicmail2)
+[![Coverage Status](https://coveralls.io/repos/github/jennizalwango/epicmail2/badge.svg?branch=ft-delete)](https://coveralls.io/github/jennizalwango/epicmail2?branch=ft-delete)
+[![Maintainability](https://api.codeclimate.com/v1/badges/b188f0d76827a1124f58/maintainability)](https://codeclimate.com/github/jennizalwango/epicmail2/maintainability)
 
-# project name
- EPIC_MAIL
+## project name 
+  epic mail
 
-## Project Overview: ##
-User sers can sign up. Users can login. Users can create groups. Users can send a message to individuals. Users can view their inbox and read messages. Users can retract sent messages. Users can save an email as draft and send it later or delete it.
+## project overview
+  Epic mail is an online system that improves and makes communcation easliy where a user 
+  Create a user account.Sign in the system.Get all received emails.
+  Get all the unread emails and get all emails sent by a the other user.
+  Get a specific user’s email.
+  Send email to individuals.Delete an email in a user’s inbox
 
-## The live demo for this project can be found at: ##
-```bash
-https://jennizalwango.github.io/EPIC_MAIL/UI/
-```
+## Prerequisties
+Inorder  to run this application you need the following;
+you need to have [python3](https://www.python.org/downloads/)  installed on your machine.
 
-## Github repository can be found at: ##
-```bash
-https://github.com/jennizalwango/EPIC_MAIL
-```
+You need to have [flask](http://flask.pocoo.org/docs/1.0/installation/) installed on your machine.
 
-## PivotalTracker link: ##
-```bash
-https://www.pivotaltracker.com/n/projects/2227254
-```
+## Installing 
 
-## Project usage: ##
-This project is divided into two parts:
-1). The user side where a user is able to create and log into the system, read their email messages, send emails and save emails
-```bash
-To access the user side use ==>>>>(email="user@gmail.com" && password="user123").
-```
+##You have to install a virutualenvirnoment, 
```

Comment:
This is not displaying as you expected the line has ## instead of formatting the file, look at this line and the one after it.